.gitignore

```diff
@@ -7,8 +7,8 @@ node_modules
 *.vsix
 **/bin/
 **/obj/
-src/modules/samples/rust/target
-src/modules/samples/rust/Cargo.lock
+**/rust/target
+**/Cargo.lock
```

Comment:
After thinking more, we may want to track `Cargo.lock` files since we are producing binaries (https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries)

Reply:
What do you think?